tests/components/statistics/test_sensor.py

```diff
@@ -251,7 +251,14 @@ async def test_sensor_defaults_binary(hass: HomeAssistant) -> None:
 
 
 async def test_sensor_source_with_force_update(hass: HomeAssistant) -> None:
```

Comment:
We could rename the test to `test_sensor_state_reported` or something, and explain in the docstring the statistics sensor handles `EVENT_STATE_REPORTED` which means we behave same also if force_update is not set by the source sensor?

Reply:
done